style.css

```diff
@@ -0,0 +1,724 @@
+body {
+  background-color: #e5e5e5;
+}
+
+.global-wrapper {
+  position: relative;
+  width: 375px;
+  height: 5410px;
```

Comment:
Kindly, I suggest you not use a static width, it is better to use percentages at 100% and make your site adapt to any device. just like height, the container better fits the content.